capstone/src/main/java/com/google/step/data/RestaurantPageViews.java

```diff
@@ -0,0 +1,51 @@
+package com.google.step.data;
+
+import java.util.List;
+
+/**
+ * Object that contains all the weekly page views of a specific restaurant.
+ */
+public class RestaurantPageViews {
+
+    private final String name;
+    private final List<WeeklyPageView> pageViews;
+
+    public RestaurantPageViews(String name, List<WeeklyPageView> pageViews) {
+        this.name = name;
```

Comment:
Make an unmodifiable copy of the pageViews here, rather than just grabbing the reference. That will make this fully thread-safe.

Reply:
Done.